content/source/docs/cloud/api/workspaces.html.md

```diff
@@ -562,6 +582,22 @@ The other refers to a workspace by its name and organization:
 | `:organization_name` | The name of the organization the workspace belongs to.                                                |
 | `:name`              | The name of the workspace to show details for, which can only include letters, numbers, `-`, and `_`. |
 
+### Workspace performance attributes
+
+The following attributes are helpful in determining the overall health and performance of your workspace configuration.  These metrics refer to the **past 30 runs that have either resulted in an error or successfully applied**.
+
+| Parameter                                  | Type   | Description                                                                                                                                                              |
+| ------------------------------------------ | ------ | --------------------------------------------------------------------------------------- |
+| `data.attributes.apply-duration-average`   | number | This is the average time runs spend in the **apply** phase, represented in milliseconds |
+| `data.attributes.plan-duration-average`    | number | This is the average time runs spend in the **plan** phase, represented in milliseconds  |
+| `data.attributes.policy-check-failures`    | number | Reports the number of run failures resulting from a policy check failure                |
+| `data.attributes.run-failures`             | number | Reports the number of failed runs                                                       |
+| `data.attributes.workspace-kpis-run-count` | number | Total number of runs taken into account by these metrics                                |
+
+### Readme
+
+The workspace resource includes a `readme` attribute.  When a workspace is VCS backed, that field is populated with the contents of the repository `README.md` file.  The contents of this field is not filtered or sanitized.  Terraform Cloud does this sanitization on the client-side.
```

Comment:
```suggestion
The workspace resource includes a `readme` attribute.  When a workspace is VCS backed, that field is populated with the contents of the repository `README.md` file.  The contents of this field are not filtered or sanitized.  Terraform Cloud does this sanitization on the client-side.
```

Also -- given that there's a ton of endpoints here that return workspace objects (notably list, but also the ones that create or modify a workspace), it feels weird to put this general information under only the "show" endpoint. It might make more sense to create a new top-level heading for these at the start of the page, before "## Create a workspace". (Suggestion: "Workspace Attributes", or something like that.)